elyra/kfp/operator.py

```diff
@@ -185,14 +190,14 @@ def __init__(
                 f"curl {common_curl_options} -L {self.bootstrap_script_url} --output bootstrapper.py && "
                 f"echo 'Downloading {self.requirements_url}' && "
                 f"curl {common_curl_options} -L {self.requirements_url} --output requirements-elyra.txt && "
+                f"curl {common_curl_options} -L {ELYRA_REQUIREMENTS_URL_PY37} --output requirements-elyra-py37.txt && "
```

Comment:
We should log the download in case it triggers an error.

```suggestion
                f"echo 'Downloading {ELYRA_REQUIREMENTS_URL_PY37}' && "
                f"curl {common_curl_options} -L {ELYRA_REQUIREMENTS_URL_PY37} --output requirements-elyra-py37.txt && "
```